### ISSUE TITLE
5.1 Reset running timer throws error

### ISSUE THREAD

**Issue by SnekCode** [midnight-studios/obs-lua#10]
Seems you have to pause the count up timer first then reset.

```
[StopWatch-[5.1].lua] Failed to call timer_cb for timer_call: attempt to call a nil value
[StopWatch-[5.1].lua] Failed to call timer_cb for timer_call: attempt to call a nil value
```
Attached Debug log:

[-Debug Log[2023-08-13_11.37.17].txt](https://github.com/midnight-studios/obs-lua/files/12329947/-Debug.Log.2023-08-13_11.37.17.txt)

**Comment by SnekCode** [comment]
Ok a little more context:
Using hot keys this error pops up even if i pause first.  But only the second time I pause.  Using the Buttons exposed in the script window I have the same behavior.  The error seems to be related to the pause feature and not just pressing reset with a running timer.

**Comment by midnight-studios** [comment]
Hey, I am aware of this issue but have not yet been able to establish how to consistently reproduce it yet. I will look into it further in hopes of resolving it. At the moment, it is more an annoyance, but does not cause a critical failure. Thanks for reporting.

**Comment by midnight-studios** [comment]
This is fixed in version 5.2
